src/tests/ftest/launch.py

```diff
@@ -1482,7 +1482,13 @@ def process_the_cores(avocado_logs_dir, test_yaml, args):
                                       "quit"])
 
             last_line = gdb_output.splitlines()[-1]
-            exe_name = last_line[7:last_line[7:].find(" ") + 7]
+            cmd = last_line[7:-1]
+            # assume there are no arguments on cmd
+            find_char = "'"
+            if cmd.find(" ") > -1:
+                # there are arguments on cmd
+                find_char = " "
+            exe_name = cmd[0:cmd.find(find_char)]
```

Comment:
Would this work?  Split on space, take the first entry and then remove and ' from the end.
```suggestion
            exe_cmd = cmd.split(' ')[0].rsplit("'")
```

Reply:
I'm not sure.  But honestly, I have too much else on my plate to keep fiddling with something that works.